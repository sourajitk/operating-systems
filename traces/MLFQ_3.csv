1,0,10
2,5,5
3,7,1
4,9,5
5,10,3
6,13,8