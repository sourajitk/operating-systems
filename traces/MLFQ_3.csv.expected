1,32
2,22
3,8
4,23
5,17
6,30